xapian-bindings/python3/docs/examples.rst

```diff
@@ -41,3 +41,33 @@ simplematchdecider.py
    :language: python
    :emphasize-lines: 12,15-18
    :linenos:
+
+.. _xapian-prepare-trainingfile:
+
+xapian-prepare-trainingfile.py
+=====================
```

Comment:
These `=` lines ought to be the same length as the line above each one (multiple instances).

Reply:
fixed in a fixup commit.